hera_mc/cm_hookup.py

```diff
@@ -466,6 +508,21 @@ def __read_hookup_cache_from_file(self):
             self.cached_hookup_dict = np.load(f).item()
             self.part_type_cache = np.load(f).item()
 
+    def show_hookup_cache_file_info(self):
+        if not os.path.exists(self.hookup_cache_file):
+            return "{} does not exist.".format(self.hookup_cache_file)
```

Comment:
The main line of this function prints information, but this case returns a string. For consistency I think this case should print the message and just return.